manifests/allow_client_on_server.pp

```diff
@@ -0,0 +1,56 @@
+# @summary Enable backup client to access backup server
+#
+# @param hostname
+#   fqdn hostname of backup client.
+#
+# @param ip
+#   ip of backup client.
+#
+# @param key
+#   The public ssh key itself; generally a long string...
+#
+# @param key_type
+#   The encryption type used in the public ssh key.
+#
+# @example
+#   profile_backup::allow_client_on_server { 'allow host hostname access to backup servers':
+#     'hostname' => String,
+#     'ip' => String,
+#     'key' => String,
+#     'key_type' => String,
+#   }
+define profile_backup::allow_client_on_server (
```

Comment:
This name is long and a tad hard to comprehend at a glance. Can we simplify?
How about just "profile_backup::allow_client"?
Or maybe move it to server, "profile_backup::server::allow_client"?

Reply:
I didn't realize we could create defined types as subclasses of other subclasses. But it seems to work and I like moving it to `profile_backup::server::allow_client`.

Reply:
renamed